README.md

```diff
@@ -56,6 +56,33 @@ Before you install and use the plug-in:
 
     For more information, see [Installing Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html).
 
+### Linux (graphical)
+
+-   Install the packages `gnome-keyring` and `libsecret`.
+
+### Linux (headless)
+
+-   Install the packages listed for graphical Linux above.
+
+-   Run the commands below to unlock the Gnome keyring. The keyring must be unlocked again after each reboot.
+
+    The second command will prompt for your password. Press Ctrl+D when you have finished typing it.
+    ```bash
+    export $(dbus-launch)
+    gnome-keyring-daemon --components=secrets --unlock
+    ```
+
+-   To automatically unlock the Gnome keyring at system boot, add the lines below to your `~/.bashrc` file.
+
+    Replace "root" with your system password.
```

Comment:
Should there be a warning here the password is being stored in plain text, or can it be assumed that's obvious here?

Reply:
I would suggest making it explicitly clear that this stores a password in plain text, and is not suggested. I still think we need to find a way to more reliably and securely make this available for headless Linux systems.

The problems are two-fold. Firstly, an X11 $DISPLAY must be available for dbus to start automatically. Secondly, the keyring needs to be unlocked once the dbus has been started. These may only affect the active connection, and may need to be run multiple times if more then one connection (such as using a remote VS Code Server via SSH or WSL).

Reply:
Updated the docs so they no longer require entering system password in plain text. The problem for remote VS Code users still exists, I suggest we create a separate issue for tracking it.